<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MixerItem</class>
 <widget class="QWidget" name="MixerItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>30</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>30</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>Click to change player name.</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="playerIndexLabel">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>1.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ClickableLabel" name="playerNameLabel">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Player Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="playerNameEdit">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="playerVolume">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Drag to adjust volume.</string>
     </property>
     <property name="maximum">
      <number>127</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickInterval">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDial" name="playerPan">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Drag to adjust panning.</string>
     </property>
     <property name="maximum">
      <number>127</number>
     </property>
     <property name="pageStep">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ClickableLabel" name="playerTuning">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Click to adjust tuning.</string>
     </property>
     <property name="text">
      <string>Tuning</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="removeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Remove Player</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
